cli/internal/runsummary/task_summary.go

```diff
@@ -1,12 +1,17 @@
 package runsummary
 
 import (
-	"github.com/vercel/turbo/cli/internal/cache"
 	"github.com/vercel/turbo/cli/internal/fs"
 	"github.com/vercel/turbo/cli/internal/turbopath"
 	"github.com/vercel/turbo/cli/internal/util"
 )
 
+type TaskCacheSummary struct {
+	Local     bool `json:"local"`
+	Remote    bool `json:"remote"`
+	TimeSaved int  `json:"timeSaved"`
+}
```

Comment:
I would like to just use `spacesCacheStatus` here, but that would change the `--dry=json` `cache` output.